docs/source/contributing_playbooks.rst

```diff
@@ -0,0 +1,92 @@
+Contributing Playbooks
+----------------------
+
+Addition of new playbooks to the edpm-ansible collection is easy,
+thanks to specialized development tooling and CI pipeline.
+However, certain requirements must be met by any new playbook,
+in order to ensure quality of the collection and a smooth operation
+of software consuming it.
+
+These requirements, outlined in the following document, go beyond
+syntactic and functional correctness of the code.
+
+Where there isn't an explicit requirement, or recommendation,
+provided by this document, the primary `Ansible documentation`_ is considered
+to be the source of truth. When this document and official Ansible docs
+are in substantial conflict, an issue should be raised.
+
+Playbook design principles
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Playbooks tie multiple roles and tasks together into cohesive whole.
```

Comment:
Generally, yes. But for the purposes of edpm-ansible I'd prefer that all tasks are in roles and playbooks only `import_role`. A quick check of all existing playbooks show this to be the case. Could we make this a rule, and only consider exceptions as needed?

Reply:
I suppose that is a valid approach.